src/main/java/com/deavensoft/paketomat/email/EmailController.java

```diff
@@ -0,0 +1,58 @@
+package com.deavensoft.paketomat.email;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@RestController
+public class EmailController {
+
+    @Autowired
+    private EmailService emailService;
+    
+
+    // Sending a simple Email
+    @PostMapping("/sendMail")
+    @Operation(summary = "sending basic mail")
```

Comment:
The first letter should be capitalized.  

- [ ] Change sending basic mail into Sending basic mail